src/app/screens/BookDetail/components/BookCard/index.tsx

```diff
@@ -25,28 +26,36 @@ export default function BookData({ data }: BookProps) {
         />
         <span className={styles.backText}>{i18next.t('BookCard:back')}</span>
       </div>
-      <div className={`${styles.bookCard} row`}>
-        <img src={data?.imageUrl} alt={i18next.t('BookCard:bookAlt') as string} className={styles.bookImg} />
-        <img src={Badge} alt={i18next.t('BookCard:badgeAlt') as string} className={styles.badge} />
-        <div className={`${styles.bookInfo} column`}>
-          <div className={styles.titleWrapper}>
-            <span className={styles.bookTitle}>{data?.title}</span>
-            <span className={styles.bookGenre}>{data?.genre}</span>
-          </div>
-          <div className={styles.bookDetails}>
-            <span className={styles.bookDetailsIndex}>{i18next.t('BookCard:author')}</span>
-            <span className={styles.bookDetailsItem}>{data?.author}</span>
-          </div>
-          <div className={styles.bookDetails}>
-            <span className={styles.bookDetailsIndex}>{i18next.t('BookCard:editor')}</span>
-            <span className={styles.bookDetailsItem}>{data?.editor}</span>
-          </div>
-          <div className={styles.bookDetails}>
-            <span className={styles.bookDetailsIndex}>{i18next.t('BookCard:publishYearh')}</span>
-            <span className={styles.bookDetailsItem}>{data?.year}</span>
+      {data ? (
```

Comment:
nice fix, but you should have loading state in mind, because it will show book not found while loading